.github/workflows/full-build.yml

```diff
@@ -102,7 +102,7 @@ jobs:
           key: ${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}-41
       - name: Install Dependencies
         id: install
-        uses: ./.github/actions/install
+        uses: ./.github/actions/install  # zizmor: ignore[unpinned-uses]
```

Comment:
Hi @amolenaar, this was a bug in zizmor. The main branch has the version pinned until they get a new release out of you want to rebase this.

Reply:
Thanks. I'll rebase on main then, and discard the last commit